### ISSUE TITLE
Tried like in the README.md described, but getting error "env: node\r: No such file or directory"

### ISSUE THREAD

**Issue by Sharaal** [RedpointGames/pkgsign#1]
Hi,

I tried out pkgsign, but after install it via `npm i -g pkgsign` and after install `keybase` I get an error:
```
pkgsign sign .
env: node\r: No such file or directory
```

If it helps to find out what can be a problem, I'm in a project directory:
```
ls
DEPENDENCIES.md README.md       index.js        package.json
```
And it's a MacOS.

**Comment by ilesinge** [comment]
I can confirm on Linux.
This seems to be due to dos end-of-lines. (\r)
You can remove them until this is resolved:
```
readlink -f `which pkgsign`
dos2unix {result of previous line}
```

**Comment by hach-que** [comment]
Hmm, okay. My guess here is that `yarn pack` / `yarn publish` (and the npm equivalents) don't convert CRLF before pushing a package into the repository. My development environment is Windows.

I'll see if I can get a fix out for this alongside issue #3.

**Comment by hach-que** [comment]
@ilesinge @Sharaal pkgsign 0.1.3 is now available in the npm repository, and you should be able to run it from the command-line now.

Again, our official advice is to install it from the GitHub repository as otherwise you can't tell pkgsign hasn't been modified by npm.
